builder/main.go

```diff
@@ -12,7 +12,7 @@ func main() {
 	r.GET("/", controllers.PingController)
 	r.GET("/verifyCheckSum", controllers.VerifyCheckSumController)
 	r.GET("/downloadFiles", nil)
-	r.GET("/buildImage", nil)
+	r.GET("/buildImage", controllers.MakeDockerImage)
 
 	// Starting the server
 	r.Run(":6969")
```

Comment:
if u ever change the port here then make the necessary updates in dev, prod docker files and the dev, prod compose files. right now the docker-compose.yaml and builder.Dockerfile still exposes 5000. update that